### DISCUSSION TITLE
Redirect email to worker

### DISCUSSION THREAD

**Discussion by clydieq** [cerb/cerb-release#71] [Q&A] [answered]
We get email as tickets to xyz@xyz.com with the subject 'Processed Drops'. How would I set these tickets to cc one of our workers? Or is there a better way to notify the worker?

**Comment by jstanden** [comment, marked as answer]
Hi @clydieq!

If you use something like Slack, Discord, or Teams you could @mention workers when tickets arrive in specific buckets:
https://github.com/cerb/cerb-release/discussions/15-Slack-notifications

Workers can also receive a copy of certain messages and reply from an external email account (back through Cerb):
https://cerb.ai/guides/mail/relaying/

They can use special `#commands` to add comments, change status, etc.

If you just want to send them a notification by email, you can use an automation:
https://github.com/cerb/cerb-release/discussions/23-sending-email-from-automations

Or you can add them as a watcher using a behavior or automation when tickets are created, and use the Notifications Emailer plugin (Setup->Plugins).
